### PR TITLE
Unholy DK: initial DF support

### PR DESCRIPTION
**Pull request by alexanderjkremer**
Sanity check performed on a raid from `retail` on 2022-12-15: https://www.warcraftlogs.com/reports/D7gTwL9HVAZdchRB

### CODE REVIEW

File: src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx [hunk 1]
```diff
@@ -318,84 +318,6 @@ class Abilities extends CoreAbilities {
...
-        cooldown: 120,
-        gcd: {
-          base: 1500,
-        },
-        castEfficiency: {
-          suggestion: true,
-          recommendedEfficiency: 0.9,
-        },
-        enabled: false,
-      },
```
Comment (emallson) [inline src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx]: this is a talent now

**Review by emallson — CHANGES_REQUESTED** [review]
great start! one small change and then we can merge

File: src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx (hunk 1 above)
Comment (alexanderjkremer) [reply src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx]: Aw - knew I'd miss something. Fixed 😄

File: src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx [hunk 2]
```diff
@@ -318,6 +317,19 @@ class Abilities extends CoreAbilities {
...
+        category: SPELL_CATEGORY.COOLDOWNS,
+        cooldown: 120,
+        gcd: {
+          base: 1500,
+        },
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.9,
+        },
+        enabled: false,
```
Comment (emallson) [inline src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx]: one more tweak

```suggestion
        enabled: combatant.hasTalent(talents.ABOMINATION_LIMB_TALENT),
```

Comment (alexanderjkremer) [reply src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx]:
```typescript
  /**
   * Whether the spell is enabled (available to the player) and should be
   * displayed. This should only be used for hiding spells that are
   * unavailable, for example due to talents.
   */
  enabled?: boolean;
```

Understood - thank your for pointing that out. I'll keep that in mind during future conversions😅

**Pull request merged**